services/federator/test/unit/Test/Federator/Client.hs

```diff
@@ -181,6 +214,41 @@ testResponseHeaders = do
     Left err ->
       assertFailure $
         "Unexpected error while connecting to mock federator: " <> show err
-    Right (status, headers, _) -> do
-      status @?= HTTP.status200
-      lookup "X-Foo" headers @?= Just "bar"
+    Right resp -> do
+      responseStatusCode resp @?= HTTP.status200
+      lookup "X-Foo" (toList (responseHeaders resp)) @?= Just "bar"
+
+testStreaming :: IO ()
+testStreaming = withInfiniteMockServer $ \port -> do
+  let req = HTTP2.requestBuilder HTTP.methodPost "test" [] mempty
+  withHTTP2Request Nothing req "127.0.0.1" port $ \resp -> do
+    let expected = mconcat (replicate 512 "Hello\n")
+    actual <- takeSourceT (fromIntegral (LBS.length expected)) (responseBody resp)
+    putStrLn "go!"
```

Comment:
I suppose this print statement is a leftover from debugging.

Reply:
Yes, thanks for catching this.